src/drawn-set.tsx

```diff
@@ -24,6 +24,10 @@ interface Props {
 
 function ChartList() {
   const charts = useDrawing((d) => d.charts);
+  const sortByLevel = useConfigState((s) => s.sortByLevel);
+  if (sortByLevel) {
+    charts.sort((a, b) => a.level - b.level);
+  }
```

Comment:
I don't want this particular sorting to happen in the component responsible for displaying the drawings. The order of cards in a drawing may start out sorted, but may also change as players take action on them.

Instead, look in `card-draw.ts` near the end. You'll find that after all the logic of picking the cards it shuffles them as a final step before returning. This new feature should be sorting by difficulty there, at the moment a new set is drawn.

Reply:
Should be updated! Thanks for the review! @noahm